src/MarianInterface.cpp

```diff
@@ -117,9 +117,14 @@ MarianInterface::MarianInterface(QObject *parent)
                     model = std::make_shared<marian::bergamot::TranslationModel>(modelConfig, std::move(bundle), modelChange->settings.cpu_threads);
                 } else if (input) {
                     if (model) {
+                        // Remove the "<!DOCTYPE html>" bit
+                        auto begin = input->find("<html");
+                        input->erase(0, begin);
```

Comment:
Any reason why [we](https://github.com/jerinphilip/tagtransfer/blob/558ff384ef8452b3abb7a64b3dd917149f7a0abf/tagtransfer/html_translator.py#L21-L29) shouldn't be doing this at bergamot-translator? This is nice to have for everyone trying to translate HTML with our library, is it not?

Reply:
You're right, we probably should. [It's not difficult to parse](https://html.spec.whatwg.org/multipage/syntax.html#the-doctype). But then we also have to support adding it back into the translated output, right?